src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,83 @@
+package step1;
+
+public class Calculator {
+
+    public Calculator(){
+
+    }
+    //더하기
+    private int _sum(int a, int b) {
+        return a + b;
+    }
+
+    //뺄셈
+    private int _minus(int a, int b) {
+        return a - b;
+    }
+
+    //곱셈
+    private int _multiply(int a, int b) {
+        return a * b;
+    }
+
+    //나눗셈
+    private int _divide(int a, int b) {
+        return (int) (a / b);
+    }
+
+
+    private int _changeInt(String numStr){
```

Comment:
변수 명에 String과 같은 타입, 자료형을 네이밍에 넣는 것은 개인적으로 권장 드리지 않습니다 :)
변수 명에 자료형을 쓰지 않아도 타입을 통해 충분히 어떤 변수인지 파악이 가능하다고 생각하며
만약 타입 변경시 변수 명도 변경해야되는 관리 포인트가 증가하는 단점도 존재한다고 생각합니다! ㅎㅎ

```suggestion
    private int _changeInt(String number) {
```